### PR TITLE
Lcg/chefstyle2

### PR DESCRIPTION
**Pull request by lamont-granquist**

### CODE REVIEW

**Comment by lamont-granquist** [comment]
@chef/client-core this makes ohai clean for chefstyle 0.3.0 that I want to release today (current chefstyle master, which was last updated a few days ago).

can i just say the networking plugin and its specs are terribad?

File: spec/unit/plugins/windows/cpu_spec.rb [hunk 1]
```diff
@@ -66,7 +66,6 @@
     @processors = [{ "description" => "Intel64 Family 6 Model 70 Stepping 1",
                      "deviceid" => "CPU0",
                      "family" => 2,
-                     "l2cachesize" => 0,
```
Comment (tas50) [inline spec/unit/plugins/windows/cpu_spec.rb]: ?

**Comment by lamont-granquist** [comment]
And the fixes in 1a882ccad96fdfc5de234c9c72c35bef1a6baba1 is all stuff that has crept in the past few days since the previous chefstyle PR to ohai.  If we merge this and then I release chefstyle that'll start enforcing stuff like double-quotes on this repo.  Right now we're not getting that, so master is accumulating more entropy over time...

File: spec/unit/plugins/windows/cpu_spec.rb (hunk 1 above)
Comment (lamont-granquist) [inline spec/unit/plugins/windows/cpu_spec.rb]: look at the end of that hash

Comment (lamont-granquist) [inline spec/unit/plugins/windows/cpu_spec.rb]: Lint/DuplicatedKey is the cop that gets raised =)

File: lib/ohai/plugins/aix/virtualization.rb [hunk 2]
```diff
@@ -82,8 +82,8 @@
               value = {
                         "mountpoint" => splat[0],
                         "device"     => splat[1],
-                        "vfs"     => splat[2],
-                        "options"     => splat[3].split(","),
+                        "vfs" => splat[2],
+                        "options" => splat[3].split(","),
```
Comment (danielsdeleo) [inline lib/ohai/plugins/aix/virtualization.rb]: do we want to tweak this to line up nicely with the values above?

**Comment by thommay** [comment]
:+1:

**Comment by danielsdeleo** [comment]
:+1: would be nice to replace the multi-line conditionals with predicate helper methods but not a big deal.

File: lib/ohai/plugins/aix/virtualization.rb (hunk 2 above)
Comment (lamont-granquist) [inline lib/ohai/plugins/aix/virtualization.rb]: i just kinda want to get it in for now.

i suspect there's another cop to turn on that will autofix all of this for us...

**Pull request merged**